packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -487,16 +490,16 @@ class AnimationController extends Animation<double>
   /// The most recently returned [TickerFuture], if any, is marked as having been
   /// canceled, meaning the future never completes and its [TickerFuture.orCancel]
   /// derivative future completes with a [TickerCanceled] error.
-  TickerFuture fling({ double velocity = 1.0, AnimationBehavior animationBehavior}) {
+  TickerFuture fling({ double velocity = 1.0, AnimationBehavior animationBehavior }) {
     _direction = velocity < 0.0 ? _AnimationDirection.reverse : _AnimationDirection.forward;
     final double target = velocity < 0.0 ? lowerBound - _kFlingTolerance.distance
                                          : upperBound + _kFlingTolerance.distance;
     double scale = 1.0;
     final AnimationBehavior behavior = animationBehavior ?? this.animationBehavior;
-    if (SemanticsBinding.instance.disableAnimations) {
+    if (_ticker.disableAnimations) {
       switch (behavior) {
         case AnimationBehavior.normal:
-          scale = 200.0;
+          scale = 200.0; // TODO(jonahwilliams): determine a better process for setting velocity.
```

Comment:
You might add a sentence to this TODO describing what's bad about the existing process, or what you'd rather see, to make it easier to know when the TODO is no longer needed.

Reply:
Done